src/hello/components/Hello.tsx

```diff
@@ -3,8 +3,8 @@ import { observer } from 'mobx-react'
 import helloStore from '../store'
 import * as css from './Hello.css'
 
-interface Props {}
-interface State {}
+type Props = {}
+type State = {}
```

Comment:
Typescript doesn't seem to like empty interfaces, and we decided to use `type` for these if they don't need to extend/be extended.